src/components/app/AuthenticatedPage.jsx

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Cookies from 'universal-cookie';
+import { LoginRedirect } from '@edx/frontend-enterprise-logistration';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { getConfig } from '@edx/frontend-platform/config';
 
 import { EnterprisePage } from '../enterprise-page';
 import { EnterpriseBanner } from '../enterprise-banner';
 import { Layout } from '../layout';
-import { LoginRedirect } from '../login-redirect';
 
 export default function AuthenticatedPage({ children, useEnterpriseConfigCache }) {
+  const config = getConfig();
+  const user = getAuthenticatedUser();
+
+  if (!user) {
+    // if user is not authenticated, remove cookie that controls whether the user will see
+    // the integration warning modal on their next visit. the expected behavior is to only
+    // see the modal once per authenticated session.
+    const cookies = new Cookies();
+    cookies.remove(config.INTEGRATION_WARNING_DISMISSED_COOKIE_NAME);
+  }
```

Comment:
These lines replicates functionality that existed in the original `LoginRedirect` component that was moved into the `@edx/frontend-enterprise-logistration` NPM package.

This functionality was kept separate from the shared component as only the Learner Portal requires deletion of this cookie.